test/messaging/components/MoveToOption.unit.spec.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import SkinDeep from 'skin-deep';
+import { expect } from 'chai';
+
+import MoveToOption from '../../../src/js/messaging/components/MoveToOption.jsx';
+
+const props = {
+  folderName: 'Folder',
+  folderId: 1,
+};
+
+describe('<MoveToOption>', () => {
+  it('should render correctly', () => {
+    const tree = SkinDeep.shallowRender(
+      <MoveToOption {...props}/>
+    );
+
+    expect(tree.getRenderOutput()).to.exist;
+  });
+
+  it('should render the expected input field', () => {
+    const tree = SkinDeep.shallowRender(
+      <MoveToOption {...props}/>
+    );
+    expect(tree.subTree('input[name=messagingMoveToFolder]')).to.exist;
+  });
+
+  it('should render the expected label text', () => {
+    const tree = SkinDeep.shallowRender(
+      <MoveToOption {...props}/>
```

Comment:
Nitpick: Since there aren't additional props to make the line too long, generally it should be okay to have these in one line.